packages/jira-adapter/src/filters/field_configuration/replace_field_configuration_references.ts

```diff
@@ -98,7 +82,7 @@ const filter: FilterCreator = ({ config, elementsSource }) => ({
       .filter(isInstanceElement)
       .filter(instance => instance.elemID.typeName === FIELD_CONFIGURATION_TYPE_NAME)
       .filter(instance => instance.value.fields !== undefined)
-      .forEach(async instance => replaceFromMap(instance, elementsSource))
+      .forEach(async instance => replaceFromMap(instance))
```

Comment:
continuing from above - 
1. can this become sync?
2. can this just be
```suggestion
      .forEach(replaceFromMap)
```
(and can we remove `awu` here as well)?